docx/section.py

```diff
@@ -7,6 +7,8 @@
 from __future__ import absolute_import, print_function, unicode_literals
 
 from collections import Sequence
+from .shared import lazyproperty
+from .header import Header
```

Comment:
Includes are alphabetical by module name (e.g. header comes before shared). Also a convention for this library is to put a blank line between the __future__ bit and the rest of the includes, then also separate Python library module includes from "local" includes by a blank line.